MovieApp/PresentationLayer/Components/OptionBar/OptionBar+Design.swift

```diff
@@ -0,0 +1,100 @@
+import UIKit
+
+extension OptionBar: ConstructViewsProtocol {
+
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    func createViews() {
+        scrollView = UIScrollView()
+        addSubview(scrollView)
+
+        contentView = UIView()
+        scrollView.addSubview(contentView)
+
+        optionButtonStack = UIStackView()
+        contentView.addSubview(optionButtonStack)
+
+        for id in 0..<10 {
+            let button = UIButton()
+            button.tag = id
+            button.addTarget(self, action: #selector(onCategorySelection), for: .touchUpInside)
+            optionButtonStack.addArrangedSubview(button)
+        }
+    }
+
+    func styleViews() {
+        scrollView.showsHorizontalScrollIndicator = false
+
+        optionButtonStack.axis = .horizontal
+        optionButtonStack.alignment = .center
+        optionButtonStack.spacing = 22
+
+        for subview in optionButtonStack.arrangedSubviews {
+            guard let button = subview as? UIButton else { return }
```

Comment:
Did you mean: `continue`?

Reply:
I felt return was appropriate because if one of the views in the stack wasn't a button the view wouldn't make sense anymore which would mean that something isn't right.